f3/granite.go

```diff
@@ -44,6 +45,24 @@ type GMessage struct {
 	Ticket Ticket
 	// Signature by the sender's public key over Instance || Round || Step || Value.
 	Signature []byte
+
+	Evidence AggEvidence
+}
+
+// Aggregated list of GossiPBFT messages with the same instance, round and value. Used as evidence for justification of messages
+type AggEvidence struct {
```

Comment:
I think either `Evidence` or `Justification` would be better names here. It's always aggregated. Calling it `justification` would align with all the uses of that word throughout the code.

Reply:
Used `Justification` in the end